pymc/sampling.py

```diff
@@ -2277,14 +2277,14 @@ def draw(
 
     Parameters
     ----------
-    vars : Variable or iterable of Variable
+    vars : RandomVariable or iterable of RandomVariable
```

Comment:
But here only variables whose Op is RandomVariable is valid? Is that right? Or any TensorVariable would work?

If the RandomVariable Op is needed, we should then update the description or use a glossary definition as "type" like we do for `tensor_like` to explain this.

Reply:
It doesn't need to be RandomVariables, any graph that can be evaluated without inputs is fine. But the purpose is indeed RandomVariables...

Reply:
It looks like we should use TensorVariable, thanks for the review.

In case it helps with the context or update some docs, I was under the impression when writing this above that: all RandomVariables are TensorVariables, but not all TensorVariables are RandomVariables, and that this function didn't work on just any TensorVariable, only on this RandomVariables subset of TensorVariables.

Reply:
I understand the impression, all examples are like that. The simplest exception is any deterministic that depends on RVs like `at.exp(at.random.normal)`, which is a perfectly valid function output.